Models/Project/Experiences.cs

```diff
@@ -0,0 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace rexfinder_api.Models;
+
+public class Experiences
+{
+    [Required]
+    public int ExperienceId { get; set; }
+
+    [Required]
+    public int UserId { get; set; }
+    public UserV1 User { get; set; }
+
+    [Required]
+    public string ExperienceTitle { get; set; }
+    public string ExperienceNotes { get; set; }
+    public bool Completed { get; set; }
+    public string ExperienceUserLocation { get; set; }
+
```

Comment:
Do we want to add a default CreatedOn date?

Reply:
Are we actually using it anywhere?